roles/bind-crc-apb/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Retrieve {{ _apb_provision_creds.SERVICE_NAME }} service configuration
+  shell: oc get configmaps/{{ crc_mobile_config_name }} -n {{ namespace }} -o jsonpath='{.data.config}'
+  register: svc_json_str_config
+
+- name: Validate CRC JSON config
+  set_fact: svc_json_config={{ svc_json_str_config.stdout | from_json }}
+  register: crc_json_config_valid
+  ignore_errors: True
+
+- fail:
+    msg: |
+      Failed to validate provided JSON configuration
+      CRC with invalid JSON config can not be created
+  when: crc_json_config_valid|failed
+
+- name: Creating CRC {{ CLIENT_ID }} configmap
+  k8s_v1_config_map:
+    name: '{{ CLIENT_ID | lower }}'
+    namespace: '{{ namespace }}'
+    annotations:
+      configType: '{{ crc_mobile_config_type }}'
+    labels:
+      name: '{{ _apb_provision_creds.SERVICE_NAME }}'
+      mobile: '{{ crc_mobile_svc_label }}'
+      serviceName: '{{ crc_mobile_svc_name }}'
+      mobileClientId: '{{ CLIENT_ID }}'
```

Comment:
we should label this with the serviceInstanceID also https://github.com/aerogearcatalog/keycloak-apb/pull/62/files#diff-e9821aec58d768e751704303ab5a0875R143

Reply:
labeled with serviceInstanceID